src/components/ModalEnterPhone.tsx

```diff
@@ -0,0 +1,97 @@
+import { connected } from 'process';
+import React,  {FC, useEffect, useState} from 'react';
+import './ModalEnterPhone.scss';
+
+interface CardProps {
+  active: boolean;
+  setActive: (bool: boolean) => void ;
+}
+
+const ModalEnterPhone: FC<CardProps> = 
+  ({
+    active, 
+    setActive,
+  }) => {
+  
+  const [phone, Phone] = useState<string>('');
+  const [phoneDirty, PhoneDirty] = useState<boolean>(false);
+  const [phoneError, PhoneError] = useState<string>('The input field must be filled');
+  const [formValid, setFormValid] = useState<boolean>(false);
+
+  useEffect(() => {
+    if (phoneError) {
+      setFormValid(false);
+    } else {
+      setFormValid(true);
+    }
+  }, [phoneError])
+
+  const sendphone = () => {
+    setActive(false);
+  }
+
+  const blurHandler = () => {
+    PhoneDirty(true);
+  }
+
+  const phoneHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const re = /^(\+375|80)(29|25|44|33)(\d{3})(\d{2})(\d{2})$/;
```

Comment:
i think you can use "React-Phone-Input-2'

Reply:
But it work